src/py/mat3ra/notebooks_utils/token_store.py

```
@@ -0,0 +1,49 @@
+    if data and data.get("expires_at", 0) > time.time() + _EXPIRY_BUFFER:
+        return data
+    return None
+
+
+def _read_cache() -> dict:
+    if _USE_BROWSER:
+        return json.loads(js.localStorage.getItem(_LOCAL_STORAGE_KEY) or "{}")
+    try:
+        with open(_FILE_PATH) as f:
+            return json.load(f)
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)